service/matching/pri_task_reader.go

```diff
@@ -170,7 +165,7 @@ func (tr *priTaskReader) completeTask(task *internalTask, res taskResponse) {
 
 	// use == so we just signal once when we cross this threshold
 	// TODO(pri): is this safe? maybe we need to improve this
-	if tr.loadedTasks == tr.backlogMgr.config.GetTasksBatchSize()/reloadFraction {
+	if tr.loadedTasks == tr.backlogMgr.config.GetTasksReloadAt() {
 		tr.Signal()
```

Comment:
On a completely unrelated note; do you think there's a better function name? While I now know what it means, `Signal` is quite vague. Since we're using `wake` for task matching, that would be an option (although it's still a bit vague). Or maybe more explicit such as `RequestLoadingTasks`.

Reply:
Signal works for me, but I've read it a bunch so :shrug:  I wonder if that name is too specific, it's more like "wake up and apply some logic to figure out if you should load more tasks"

Reply:
`SignalTaskLoading`?